pkg/morph/client/constructor.go

```diff
@@ -105,12 +106,6 @@ func New(key *keys.PrivateKey, endpoint string, opts ...Option) (*Client, error)
 		waitInterval: cfg.waitInterval,
 	}
 
-	if len(cfg.notaryOpts) != 0 {
```

Comment:
Should we remove `cfg.notaryOpts` then?

Reply:
looks like it is useless then

deleted